src/fmod_studio.rs

```diff
@@ -2,6 +2,7 @@ use std::env::var;
 use std::fs::canonicalize;
 use std::path::{Path, PathBuf};
 
+use crate::components::audio_source::AudioSource as BevyFmodAudioSource;
```

Comment:
Had a naming conflict with AudioSource. Feel free to handle this differently or call change the alias.

Reply:
I don't seem to have this problem with just `use crate::components::audio_source::AudioSource;`, please try again :P

Reply:
You are correct, I am a fool.